pkg/executor/executortype/newdeploy/newdeploymgr.go

```diff
@@ -132,7 +133,7 @@ func MakeNewDeploy(
 		useIstio:               enableIstio,
 
 		defaultIdlePodReapTime:     2 * time.Minute,
-		objectReaperIntervalSecond: time.Duration(getObjectReaperInterval(logger, 5)) * time.Second,
+		objectReaperIntervalSecond: time.Duration(executorUtils.GetObjectReaperInterval(logger, fv1.ExecutorTypeContainer, 5)) * time.Second,
```

Comment:
fv1.ExecutorTypeContainer, please change it to fv1.ExecutorTypeNewdeploy

Reply:
Fixed